### DISCUSSION TITLE
How can I display files from a related entity on my show operation? It uses "upload_multiple"

### DISCUSSION THREAD

**Discussion by NuktukDev** [Laravel-Backpack/community-forum#1246] [Q&A (Help)]
**Important Info: I use spatie media library**

I have two entities that are related to each other.  Job and ProcessedJob.

On the "Job" show operation, I want to show a list of of related ProcessedJob's and their attributes, as well as the Job entity attributes underneath.

I'm currently doing this manually with a widget and a loop, but I can't get the ProcessedJob's "upload_multiple" column to display the files.

I think it's because the entity is set on "Job". How can I do this?

Here's my hacky way so far:
```
foreach($this->crud->getCurrentEntry()->processedJobs as $processedJob) {
                CRUD::column('engineer_images')->label('Engineer Images')->type('upload_multiple')->withMedia();
                $body = view('crud::columns.upload_multiple', [
                    'entry' => $processedJob,
                    'crud' => $this->crud,
                    'column' => $this->crud->columns()['engineer_images']
                ])->render();
```

Note: The "engineer_images" is not a column in the DB.

[a comment by an automated account is not shown]

**Reply by NuktukDev** [reply]
"engineer_images" is not a db column. I use spatie media.

**Comment by pxpm** [comment]
Hey @NuktukDev It should probably be done through the `relationship` column, defining the `subfields`. 
```php
CRUD::column('processedJobs')->subfields([
  [
      'name' => 'engineer_images',
      'type' => 'upload_multiple',
      'withMedia' => true,
  ],
]);
```

Haven't tested it tho, there were some issues regarding relationships in uploaders that we are fixing in v7 (crud) v2 (media-library).

Another possible solution is creating a custom column, and add it to your crud like:
```php
CRUD::column('processedJobs.engineer_images')->type('my_custom_column');

// in your `my_custom_column` you should be able to have access to the `$column['value']`
```

Cheers

**Comment by jcastroa87** [comment]
Due to not activity, I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
